File: iOS/iOS/ViewController.m
//
//  ViewController.m
//  iOS
//
//  Created by Nirupa Parmhans on 10/5/14.
//  Copyright (c) 2014 Nirupa Parmhans. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (IBAction)buttonPressed:(UIButton *)sender

{
    self.titleLable.text = self.textField.text;
    [self.textField resignFirstResponder];
}
@end

File: Postcard/Postcard/ViewController.swift
//
//  ViewController.swift
//  Postcard
//
//  Created by Nirupa Parmhans on 10/15/14.
//  Copyright (c) 2014 Nirupa Parmhans. All rights reserved.
//

import UIKit

class ViewController: UIViewController
{

    @IBOutlet weak var messageLabel: UILabel!
    
    @IBOutlet weak var enterNameTextField: UITextField!
    
    @IBOutlet weak var enterMessageTextField: UITextField!
    
    override func viewDidLoad()
    {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning()
    {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    
    @IBAction func sendNowButton(sender: UIButton)
    {
        messageLabel.hidden = false
        
    }

}

